### DISCUSSION TITLE
'ValueError: Only supported for TrueType fonts' when performing a ProfileReport (JSON works).

### DISCUSSION THREAD

**Discussion by bnpani** [Data-Centric-AI-Community/fg-data-profiling#1357] [Q&A]
**Edit: I just tested the old library and there are no issues there.**

I have this issue when performing a ProfileReport for a Pandas dataframe. Looks like there is something wrong with the word cloud method/s, but I didn't find any config to disable/customize that part within the documentation. The strange part is that yesterday when I first tried this dataframe, the report was generated without issues, but now only the json format works. 

Data info:

 #   Column            Non-Null Count  Dtype  
---  ------            --------------  -----  
 0   ORDERNUMBER       147 non-null    int64  
 1   QUANTITYORDERED   147 non-null    int64  
 2   PRICEEACH         147 non-null    float64
 3   ORDERLINENUMBER   147 non-null    int64  
 4   SALES             147 non-null    float64
 5   ORDERDATE         147 non-null    object 
 6   STATUS            147 non-null    object 
 7   QTR_ID            147 non-null    int64  
 8   MONTH_ID          147 non-null    int64  
 9   YEAR_ID           147 non-null    int64  
 10  PRODUCTLINE       147 non-null    object 
 11  MSRP              147 non-null    int64  
 12  PRODUCTCODE       147 non-null    object 
 13  CUSTOMERNAME      147 non-null    object 
 14  PHONE             147 non-null    object 
 15  ADDRESSLINE1      147 non-null    object 
 16  ADDRESSLINE2      147 non-null    object 
 17  CITY              147 non-null    object 
 18  STATE             147 non-null    object 
 19  POSTALCODE        147 non-null    object 
 20  COUNTRY           147 non-null    object 
 21  TERRITORY         147 non-null    object 
 22  CONTACTLASTNAME   147 non-null    object 
 23  CONTACTFIRSTNAME  147 non-null    object 
 24  DEALSIZE          147 non-null    object 
dtypes: float64(2), int64(7), object(16)
memory usage: 33.9+ KB

**Comment by rzeydelis-AS** [comment]
I am also getting the same issue..

**Comment by alongriskified** [comment]
I think it's related to this
https://github.com/amueller/word_cloud/issues/729
tl;dr consider upgrading pip and pillow:

`pip install --upgrade pip`
`pip install --upgrade Pillow`

**Reply by ironavt** [reply]
I had the same issue and this worked for me, thank you!

**Reply by jhemmi** [reply]
pip and Pillow upgrade solve the problem for me
